src/wormhole/transit.py

```diff
@@ -579,6 +580,8 @@ def __init__(self,
         self._waiting_for_transit_key = []
         self._listener = None
         self._winner = None
+        if reactor is None:
+            reactor = twisted.internet.reactor
```

Comment:
This will probably fail.. if `twisted.internet` had a property named `reactor`, it would be loaded already and we'd lose the benefit of delayed loading. Maybe remove the `import twisted.internet` from the top, and replace this with `from twisted.internet import reactor`

Reply:
You're right - the code above only works because twisted sneakily removes the twisted.intenet.reactor module and replaces it with an attribute of the same name. I prefer your suggestion as it doesn't rely on this madness!